app/views/posts/index.html.erb

```diff
@@ -1,2 +1,15 @@
-<h1>Posts#index</h1>
-<p>Here is a list of posts for a given user</p>
\ No newline at end of file
+
+<div class ='user'>
+  <img class="user_img" src=<%= @user.photo %> alt=<%= @user.name %> />
+  <%= render "share/user",   user: @user  %>
+</div>
+
+<% @posts.each_with_index do |post, index| %>
+  <div class='post_details'>
+    <%= render "share/posts",   post: post %>
+
+   <% post.comments.each do |comment| %>
+      <%=render "share/comments", comment: comment%>
+   <% end %>
```

Comment:
- [ ] It will be great to return recent comments using the method you have created in the posts model. In this implementation, you are returning all the comments but at this point, it should only return the recent 5 comments

Reply:
@peterrobert . Thank you very much for reminding me and also for your effort. I really appreciate that